config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  root to: 'static#index'
```

Comment:
It's fine to change the root to static#about but you need to create a about action in the static controller

Reply:
are we changing the navbar again? I thought the one Z had up was fine.

Reply:
we arent changing it again, ive already took ine out